example-ideal.txt

```diff
@@ -0,0 +1,14 @@
+#start = corner
+N_sweep_move_W {
+  N_ -> N  // move north when there's not a wall
+  N! -> W S_sweep
```

Comment:
Yeah I agree that this is slightly hard to read code. It's very short but without words except for state names it's slightly difficult to read. With that being said, the original picobot code was also very hard to read (especially without practice), so this is definitely an upgrade.